aptos-move/aptos-vm/src/errors.rs

```diff
@@ -76,7 +77,7 @@ pub fn convert_prologue_error(
                 // in the Move repo.
                 (INVALID_STATE, EACCOUNT_NOT_MULTISIG) => StatusCode::ACCOUNT_NOT_MULTISIG,
                 (PERMISSION_DENIED, ENOT_MULTISIG_OWNER) => StatusCode::NOT_MULTISIG_OWNER,
-                (INVALID_ARGUMENT, EMULTISIG_TRANSACTION_NOT_FOUND) => {
+                (NOT_FOUND, EMULTISIG_TRANSACTION_NOT_FOUND) => {
```

Comment:
Do the other multisig errors show up correctly? for example ENOT_MULTISIG_OWNER?

Reply:
Yes, I tested and observed that it correctly produced the following error messages:
```
{
  "Error": "API error: API error Error(VmError): Invalid transaction: Type: Validation Code: ACCOUNT_NOT_MULTISIG"
}

{
  "Error": "API error: API error Error(VmError): Invalid transaction: Type: Validation Code: MULTISIG_TRANSACTION_INSUFFICIENT_APPROVALS"
}

{
  "Error": "API error: API error Error(VmError): Invalid transaction: Type: Validation Code: NOT_MULTISIG_OWNER"
}

{
  "Error": "API error: API error Error(VmError): Invalid transaction: Type: Validation Code: MULTISIG_TRANSACTION_PAYLOAD_DOES_NOT_MATCH_HASH"
}
```